build.sbt

```diff
@@ -557,7 +557,11 @@ lazy val sharedSettings = Def.settings(
     else Nil
   },
   scalacOptions ++= {
-    if (isScala213.value) List("-Xfatal-warnings", "-Wconf:cat=deprecation:is")
+    if (isScala213.value)
+      List(
+        // "-Xfatal-warnings", https://github.com/scalameta/scalameta/issues/3576
```

Comment:
I wasn't able to fix the warning, and ignoring it only with messages makes all the other warnings pop up.

Reply:
what was the warning?

Reply:
It's the ones in https://github.com/scalameta/scalameta/issues/3576

Reply:
let me give it a try first

Reply:
let me give it a try first